runtime/commonTest/src/kotlinx/serialization/features/PolymorphismTest.kt

```diff
@@ -48,6 +48,7 @@ class PolymorphismTest : JsonTestBase() {
 
     object PolyDefaultSerializer : JsonTransformingSerializer<PolyDefault>(PolyDefault.serializer(), "foo") {
         override fun readTransform(element: JsonElement): JsonElement {
+            (element.jsonObject.content as MutableMap).remove("type")
```

Comment:
I'd like to avoid such hacks in test code, because it may be an example for someone.

Reply:
Seems you forgot to delete the line after fix?